test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/ReferCaseControllerSpec.scala

```diff
@@ -65,6 +73,32 @@ class ReferCaseControllerSpec extends WordSpec with Matchers with UnitSpec
   private def controller(requestCase: Case, permission: Set[Permission]) = new ReferCaseController(
     new RequestActionsWithPermissions(permission, c = requestCase), casesService, messageApi, appConfig)
 
+  private def aMultipartFileWithParams(params: (String, Seq[String])*): MultipartFormData[TemporaryFile] = {
+    val file = TemporaryFile("example-file.txt")
+    val filePart = FilePart[TemporaryFile](key = "email", "file.txt", contentType = Some("text/plain"), ref = file)
+    MultipartFormData[TemporaryFile](dataParts = params.toMap, files = Seq(filePart), badParts = Seq.empty)
+  }
+
+  def aEmptyMultipartFileWithParams(params: (String, Seq[String])*): MultipartFormData[TemporaryFile] = {
```

Comment:
some of these are `private` some aren't, lets be consistent :)

Reply:
OK can fix this and make them all private